documentation/observers.html

```diff
@@ -1,11 +1,87 @@
 ---
 layout: largePage
 column: headers/documentation.html
-color: is-primary
+color: is-dark
 title: Observers
 subtitle: Observe behaviour of messages or commands.
 menu: menus/documentation/documentation.html
 ---
 
+<div class="content">
 
-Text about observers here.
+    <h1 class="title"> What is an Observer </h1>
+
+    <p> An Observer is a function that observes the behavior of messages or commands. They are always keeping an eye on the behavior of messages and commands dependant on what you want it to observe. </p> 
+    <p> Observers can be restricted to observe certain types of text whether it be messages, commands or both. In addition, you can choose to disable an Observer easily. </p>
+    <p> Spark already contains some pre-made Observers for you: </p>
+    <ul>
+        <li>
+            <h4 class="subtitle">prefixHelp</h4>
+            <p> When you do @bot-name, it will tell you the prefix of the bot. </p>
+        </li>
+    </ul>
+
+    <p> Below you will find out how to create an Observer, as well as the methods that may be used to help you with creating an Observer. </p>
+          
+        
+            
+        <h1 class="title"> Constructor </h1>
+        
+        <p> To create an Observer, you will need to create a new ".js" file inside the functions/observers folder. </p>
+        <p> Below is an example of how a default Observer file is structured: </p>
+        
+        <pre>
+            var Spark = require("sparkbots")
+            const Observer = Spark.observer("name")
+            
+            Observer.code = (client, message) => {
+            
+              // Execute Code Here
+            
+            }
+            module.exports = Observer;
+    </pre>
+        
+        
+        
+        <h1 class="title"> Methods </h1>
+        
+        <ul>
+          <li>
+              <p class="subtitle"> setType(<span class="argument"> Type </span>) </p>
```

Comment:
To keep things uniform, i suggest keeping all subtitles to <h4> as the styleguide suggests.

Reply:
Complete.